src/SFA.DAS.Employer.PR.Web/Authentication/EmployerAccountAuthorizationHandler.cs

```diff
@@ -36,7 +36,7 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
         return Task.CompletedTask;
     }
 
-    private bool IsEmployerAuthorised(AuthorizationHandlerContext context)
+    public bool IsEmployerAuthorised(AuthorizationHandlerContext context)
```

Comment:
Why public?

Reply:
I got stuck trying to figure out how to test this when it was private.  I figured it out now.